hash/msx1_cass.xml

```diff
@@ -1497,7 +1632,7 @@ license:CC0-1.0
 	</software>
 
 	<software name="basedatt">
-		<description>Base de Datos (Spain, Toshiba)</description>
+		<description>Base de Datos (Spain, EMSA)</description>
 		<year>198?</year>
 		<publisher>Toshiba</publisher>
```

Comment:
Is the publisher supposed to be changed to ESMA here, too?

Reply:
This is an interesting one: there were 7 different pieces of software released as "Base de Datos", 2 of those were published by Toshiba, one developed by DimensionNEW, and one by EMSA.
I know we normally put the publisher in the description but for this case I chose to put the developer there to make clear which variant this is. If you prefer to see this done in a different way then I can change it.

Reply:
You can add info elements for developers as well.  It makes sense that the published can't be used to disambiguate when it's the same in multiple cases.